README.md

```diff
@@ -9,10 +9,11 @@
 [![Access Here](https://img.shields.io/badge/Access-Here-brightgreen?style=?style=for-the-badge)](https://typing-test-project.vercel.app/)
 [![License Badge](https://img.shields.io/github/license/arpitghura/hacktoberfest?color=2b9348)](https://github.com/arpitghura/typing-test/blob/main/LICENSE)
 [![Code of Conduct](https://img.shields.io/badge/Codeof-Conduct-brightgreen?style=?style=for-the-badge)](https://github.com/arpitghura/typing-test/blob/main/CODE_OF_CONDUCT.md)
+
 [![Typing Test participated in KWoC 22](https://img.shields.io/badge/KWoC-2022-blue?style=?style=for-the-badge)](https://kwoc.kossiitkgp.org/projects?query=typing+test)
 [![Typing Test participating in DIVERSION 2023](https://img.shields.io/badge/DIVERSION-2023-blue?style=?style=for-the-badge)](https://diversion.tech/)
 [![Typing Test participating in IWOC 2023](https://img.shields.io/badge/IWOC-2023-blue?style=?style=for-the-badge)](https://iwoc.live/project)
-
+[![Discord](https://img.shields.io/badge/Discord-blueviolet?style=?style=for-the-badge)](https://discord.gg/7c3VGDutmS)
```

Comment:
Instead of this, Use this 
[![Discord](https://img.shields.io/discord/1050391184217997422?label=Join%20Discord%20&logo=discord&style=for-the-badge)](https://discord.gg/7c3VGDutmS)

```suggestion
[![Discord](https://img.shields.io/discord/1050391184217997422?label=Join%20Discord%20&logo=discord&style=for-the-badge)](https://discord.gg/7c3VGDutmS)
```